### PR TITLE
Action set Renderer 

### PR DESCRIPTION
**Pull request by rohshar6**
## Description
Created Action Set Renderer with the following properties.
 1. Vertical or Horizontal orientation based on hostconfig
 2. Alignment based on hostconfig
 3. MaxActions rendered
 4. Test cases for action set
 5. Added a new test case for container renderer

## Sample Card
Screenshot 2021-03-17 at 2 41 42 PM

Screenshot 2021-03-17 at 2 41 51 PM

Screenshot 2021-03-17 at 2 42 08 PM

### CODE REVIEW

File: source/macos/AdaptiveCards/AdaptiveCards/Views/ACRActionSetView.swift [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    }
+    
+    override func viewDidMoveToSuperview() {
+        super.viewDidMoveToSuperview()
+        guard let view = superview else { return }
+        widthAnchor.constraint(equalTo: view.widthAnchor).isActive = true
+    }
+    
+    private func customLayout() {
+        print("@running custom layout")
```
Comment (rohshar6) [inline source/macos/AdaptiveCards/AdaptiveCards/Views/ACRActionSetView.swift:57]: added this comment to let others know, custom layout is rendering

File: source/macos/AdaptiveCards/AdaptiveCards/Renderers/ActionSetRenderer.swift [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                default:
+                    actionSetView.alignment = alignment
+                }
+            }
+        }
+        
+        var accumulatedWidth: CGFloat = 0, maxWidth: CGFloat = 0
+        if actionSet.getActions().isEmpty {
+            return actionSetView
+        }
```
Comment (blue-stone99) [inline source/macos/AdaptiveCards/AdaptiveCards/Renderers/ActionSetRenderer.swift:39]:
```suggestion
        guard !actionSet.getActions().isEmpty else {
            logInfo("Actions in ActionSet is empty")
            return actionSetView
        }
```

File: source/macos/AdaptiveCards/AdaptiveCards/Renderers/ActionSetRenderer.swift [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        
+        var actionsToRender: Int = 0
+        if let maxActionsToRender = adaptiveActionHostConfig?.maxActions, let uMaxActionsToRender = maxActionsToRender as? Int {
+            actionsToRender = min(uMaxActionsToRender, actionSet.getActions().count)
+            if actionSet.getActions().count > uMaxActionsToRender {
+                print("WARNING: Some actions were not rendered due to exceeding the maximum number \(uMaxActionsToRender) actions are allowed")
```
Comment (blue-stone99) [inline source/macos/AdaptiveCards/AdaptiveCards/Renderers/ActionSetRenderer.swift:45]:
```suggestion
            if actionSet.getActions().count > uMaxActionsToRender {
                logError("WARNING: Some actions were not rendered due to exceeding the maximum number \(uMaxActionsToRender) actions are allowed")
```

**Review by blue-stone99 — COMMENTED** [review]
Minor comments. This is element is 🔥.

**Pull request merged**